DependencyInjection/Compiler/JMSTranslationPass.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Elao\Bundle\FormTranslationBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class JMSTranslationPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if (!$container->hasExtension('jms_translation')) {
```

Comment:
I'd rather the service to be defined only if the JMS Translation extension exists (and not removed if not).

What do you think?